### PR TITLE
Add BRNZ as an Asset

### PR DESCRIPTION
**Pull request by triccs**
## Description

Adding an Osmosis native token called BRNZ

### Adding Assets

If adding a new asset, please ensure the following:
- [ ] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).): (choose one)
- [ ] Ready -- A liquidity pool(s) has been created and has liquidity. The pool ID is: ______
- [ ] Soon -- A pool will be created soon. Do not merge this PR until liquidity is ready. 
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Note: This method is not recommended because the Supercharged Liquidity pool type cannot be created via the Osmosis Zone app)
- [x] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds. The StreamSwap Bootstrapping phase begins (when?): ______

### CODE REVIEW

**Comment by triccs** [comment]
Here is the PR for Cosmos Registry: https://github.com/cosmos/chain-registry/pull/4228

I didn't a date for the bootstrap but it should start on the first few days of April, I need to get the token added here first so I can start the Stream using the UI.

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -2890,6 +2890,15 @@
       "twitter_URL": "https://twitter.com/Gelotto2",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "osmosis",
+      "base_denom": "factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz",
+      "twitter_URL": "https://twitter.com/the_memebrane",
+      "osmosis_verified": true,
+      "override_properties": {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2899]: I don't see why a property override is required here??

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -2890,6 +2890,15 @@
       "twitter_URL": "https://twitter.com/Gelotto2",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "osmosis",
+      "base_denom": "factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz",
+      "twitter_URL": "https://twitter.com/the_memebrane",
+      "osmosis_verified": true,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2898]: Please add `"osmosis_unlisted": true`

**Comment by JeremyParish69** [comment]
It seems your fork also copied Actions and your PR is merging your main branch, meaning the Query Pools Action automatically added some updates to your PR, and I cannot resolve conflicts from GitHub web app due to the complexity.

**Comment by JeremyParish69** [comment]
This PR is all kinds of messed up by the conflicts caused by Actions duplicated on your branch. I can help recreate, but I'm not seeing any response and still haven't seen any streamswap.

**Comment by JeremyParish69** [comment]
Please resubmit once ready to address the comments

**Pull request closed without merging**